app/services/ethereum/bridge/service.go

```diff
@@ -35,6 +39,8 @@ type BridgeContractService struct {
 	contractInstance *bridge.Bridge
 	Client           *ethclient.EthereumClient
 	mutex            sync.Mutex
+	custodians       custodians.Custodians
+	servicefee       servicefee.Servicefee
```

Comment:
You should include the `logger` here as property similar to Consensus Message Handler f.e
We want to have a context for the logger. That's why we do getLoggerFor in other files. 
```go
config.GetLoggerFor("Bridge Contract Service")
```